workers/github_worker_base.rb

```diff
@@ -0,0 +1,112 @@
+require_relative "worker_base"
+require_relative "../shared/models/provider_credential"
+require_relative "../shared/logging_module"
+require_relative "../services/test_runner_service"
+require_relative "../services/code_hosting/git_hub_service"
+
+module FastlaneCI
+  # Base class for GitHub workers
+  class GithubWorkerBase < WorkerBase
```

Comment:
GitHub (i know we get the casing wrong in a few places too)

Reply:
good catch!